packages/rpc-api/src/__tests__/send-transaction-test.ts

```diff
@@ -224,17 +246,53 @@ describe('sendTransaction', () => {
                 { encoding: 'base64', preflightCommitment: 'processed' },
             )
             .send();
-        await expect(resultPromise).rejects.toThrow(RpcError);
         await expect(resultPromise).rejects.toThrow(
-            /Attempt to debit an account but found no record of a prior credit/,
+            new SolanaError(SOLANA_ERROR__JSON_RPC__SERVER_ERROR_SEND_TRANSACTION_PREFLIGHT_FAILURE, {
+                accounts: null,
+                cause: new SolanaError(SOLANA_ERROR__TRANSACTION_ERROR__ACCOUNT_NOT_FOUND),
+                logs: [],
+                returnData: null,
+                unitsConsumed: 0,
+            }),
         );
-        await expect(resultPromise).rejects.toMatchObject({
-            code: -32002 satisfies (typeof SolanaRpcErrorCode)['JSON_RPC_SERVER_ERROR_SEND_TRANSACTION_PREFLIGHT_FAILURE'],
+    });
+    it('fatals when the fee payer has insufficient funds to pay rent', async () => {
```

Comment:
Added this, because the _existing_ test wasn't actually testing anything about fees – only that the account _didn't exist_.